cypress/e2e/brewery/DatasetManager_e2e.cy.js

```diff
@@ -40,7 +40,7 @@ describe('End-to-end test of the dataset manager view', () => {
     DatasetManager.getIndicatorCard('bars_count').should('be.visible');
     DatasetManager.getKpiValue(DatasetManager.getIndicatorCard('bars_count')).should('have.text', 1);
 
-    DatasetManager.getAllDeleteDatasetButtons().click();
+    DatasetManager.getAllDeleteDatasetButtons().eq(0).click();
```

Comment:
I think using `eq(0)` won't guarantee that we click on the `DatasetList` button, so this change might modify the behavior of the test. We should probably use different selectors or "data-cy" tags to make sure we can select the button we want to test. `getAllDeleteDatasetButtons` was initially only retrieving the buttons in `DatasetList`, maybe we should change the data-cy value of the Overview button so it won't be selected ?

Reply:
The two buttons have different data-cy strings. The one in the dataset list is named `dataset-delete-button-<DATASETID>` and the new on in the top right is called `dataset-delete-button-DatasetActions<DATASETID>` The cypress test fails because it tries to click on both buttons at the same time as it is using the `getAllDeleteDatasetButtons` function which searches for attributes starting with `dataset-delete-button-`. To avoid the  `eq(0)` we could instead modify the data-cy of the delete button by prepending the location string.